src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -144,13 +151,23 @@ await functionMetadata
                 await FileUtility.WriteAsync(dataFilePath, functionMetadata.TestData);
             }
 
+            var updatedFunctionMetadata = (await _hostFunctionMetadataProvider.GetFunctionMetadataAsync(_languageWorkerOptions.CurrentValue.WorkerConfigs, configChanged))
+                .FirstOrDefault(metadata => Utility.FunctionNamesMatch(metadata.Name, name));
+
+            bool getFunctionMetadataSuccessful = false;
+            FunctionMetadataResponse functionMetadataResult = null;
+            if (updatedFunctionMetadata != null)
+            {
+                string routePrefix = await GetRoutePrefix(hostOptions.RootScriptPath);
+                var baseUrl = $"{request.Scheme}://{request.Host}";
+                functionMetadataResult = await updatedFunctionMetadata.ToFunctionMetadataResponse(hostOptions, routePrefix, baseUrl);
+                getFunctionMetadataSuccessful = true;
+            }
```

Comment:
Should we log any relevant state/information in the `else` case here? The API endpoint which calls this method is going to return 500 status code in that case and any useful information being logged will be helpful to debug.

Reply:
@kshyju i looked through the call stack to see if we can provide any useful information. I couldn't identify anything additional we can log here.